### PR TITLE
AC New Automated Test - LRAC-11056 | master

### PR DESCRIPTION
**Pull request by joshitagaki**
Ticket: https://issues.liferay.com/browse/LRAC-11056
Bug: https://issues.liferay.com/browse/LRAC-8557
Testray: https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1754162123&testrayRunId=1754162149

### CODE REVIEW

**Comment by joshitagaki** [comment]
ci:test:sf

**Review by christianstokes — APPROVED** [review]
Nice, looks good!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesSearchTerms.testcase [hunk 1]
```diff
@@ -74,6 +74,56 @@ definition {
...
+
+			ACTimeFilter.setLast24Hours();
+		}
+
+		task ("Assert Keywords Searched") {
+			for (var keyword : list "atenção com a criança,子供との注意,atención al niño") {
+				AssertElementPresent(
+					key_keyword = "${keyword}",
+					locator1 = "ACSites#SEARCHED_KEYWORD");
+			}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesSearchTerms.testcase:123]: Hey @joshitagaki, Your test is very good, just a little suggestion here. I think you could use `viewGenericText` macro for that case, we could involve the assert in the macro. What do you think?

**Comment by christianstokes** [comment]
@joshitagaki I can forward after you check/confirm if the viewGenericText will work or not.

**Review by ViniLopes20 — APPROVED** [review]
LGTM, thanks @joshitagaki

**Review by manuferreira — APPROVED** [review]
Thanks Josh, LGTM!

**Comment by ViniLopes20** [comment]
Hi team, Thanks for reviewing!

I'm forwarding it!

**Comment by ViniLopes20** [comment]
ci:forward

**Pull request closed without merging**